weblate/api/views.py

```diff
@@ -1971,6 +1978,23 @@ def destroy(self, request: Request, *args, **kwargs):
             )
         return Response(status=HTTP_204_NO_CONTENT)
 
+    @action(detail=True, methods=["get"])
+    def translations(self, request: Request, *args, **kwargs):
+        unit = self.get_object()
+        user = request.user
+        user.check_access_component(unit.translation.component)
+
+        if unit.pk != unit.source_unit.pk:
```

Comment:
Use a generic interface for this (which also avoids one additional SQL query): 

```suggestion
        if unit.is_source:
```